Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
> What does this refer to? If it's about all the inreplace and related stuff, we could as well ignore the install target in the Makefile and simply copy the two files that would be installed manually. 

It's more that upstream is dead and that's why this stuff won't go upstream. There's at least two of these that aren't met: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff